samples/packages/lake-formation-tag-sync/src/test/kotlin/com/atlan/pkg/lftag/LakeTagSynchronizerTest.kt

```diff
@@ -0,0 +1,216 @@
+/* SPDX-License-Identifier: Apache-2.0
+   Copyright 2023 Atlan Pte. Ltd. */
+package com.atlan.pkg.lftag
+import LakeFormationTagSyncCfg
+import com.atlan.Atlan
+import com.atlan.model.assets.Column
+import com.atlan.model.assets.Connection
+import com.atlan.model.assets.Database
+import com.atlan.model.assets.Schema
+import com.atlan.model.assets.Table
+import com.atlan.model.enums.AtlanConnectorType
+import com.atlan.model.enums.AtlanCustomAttributePrimitiveType
+import com.atlan.model.typedefs.AttributeDef
+import com.atlan.model.typedefs.CustomMetadataDef
+import com.atlan.model.typedefs.EnumDef
+import com.atlan.pkg.PackageTest
+import com.atlan.pkg.Utils
+import com.atlan.util.AssetBatch
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import com.google.common.io.Files
+import mu.KotlinLogging
+import org.testng.annotations.Test
+import java.io.File
+import kotlin.test.assertEquals
+
+private const val DATABASE_NAME = "db_test"
+
+private const val attr1 = "Security Classification"
+
+private const val attr2 = "Privacy Sensitivity"
+
+private const val attr3 = "Data Load Method"
+
+private const val CONNECTION_MAP_JSON = "connection_map.json"
+
+private const val PREFIX = "stuff"
+
+private const val METADATA_MAP_JSON = "metadata_map.json"
+
+private const val S3_BUCKET = "lakestoragetag"
+
+class LakeTagSynchronizerTest : PackageTest() {
+    override val logger = KotlinLogging.logger {}
+    private val c1 = makeUnique("lftagdb")
+    private var connectionQualifiedName = ""
+    private var tableGuid = ""
+    private var columnGuid = ""
+    private val cm1 = makeUnique("lftcm")
+    private val enum1 = makeUnique("lfenum")
+    private val mapper = jacksonObjectMapper()
+
+    private fun createConnections() {
+        Connection.creator(c1, AtlanConnectorType.REDSHIFT)
```

Comment:
I have made the change. Excuse my density, but I don't understand the concurrency concerns or why `AtlanConnectorType.MINISQL` is better.

Reply:
Connection `qualifiedName`s are unique via timestamp + connector type — since tests run in parallel (in separate JVMs) we cannot control potential conflicts on the timestamp portion (two different tests could create a connection at the same time). Hence the only direct way we can control uniqueness of the connection's `qualifiedName` across tests is to ensure that every test uses a different connector type.